src/parser/parser.ts

```diff
@@ -28,33 +28,44 @@ function wrap(contentType: string, content: string, linkPath?: string): string {
     return `<div class="content-${contentType}">${link}\n${content}</div>`;
 }
 
-function textRenderer(
-    fileEnding: string | undefined,
-): { render: Renderer; contentType: string } | undefined {
-    if (!fileEnding) return undefined;
-    if (config.mdExtensions.includes(fileEnding)) {
-        return { render: renderMarkdown, contentType: 'markdown' };
-    }
-    if (fileEnding === 'ipynb') {
-        return { render: renderNotebook, contentType: 'ipynb' };
+export function renderBody(
+    path: string,
+    mime: string,
+    // undefined to read from file, string for live reloading
+    content: string | undefined = undefined,
+): string | undefined {
+    // render image file
+    if (mime.startsWith('image/')) {
+        return wrap(
+            'image',
+            renderMarkdown(`${pathHeading(path)}\n\n![${path}](./${pbasename(path)})\n`),
+            pdirname(path),
+        );
     }
-}
 
-export const shouldRender = (mime: string): boolean =>
-    mime.startsWith('text/') || mime === 'application/json' || mime === 'inode/x-empty';
+    // render text files
+    if (mime.startsWith('text/') || mime === 'application/json' || mime === 'inode/x-empty') {
```

Comment:
```suggestion
    if (
        mime.startsWith('text/') ||
        mime === 'application/javascript' ||
        mime === 'application/json' ||
        mime === 'inode/x-empty'
    ) {
```

Looks like this works (with added `application/javascript`), just confusing that viewing JS worked without this.